main.go

```diff
@@ -307,7 +307,7 @@ func validateParams() providerConfig {
 	var ecrEnabled bool
 
 	awsAccountID = os.Getenv("awsaccount")
-	if len(awsAccountID) == 0 {
+	if len(awsAccountID) == 0 || awsAccountID == "changeme" {
```

Comment:
That is a great idea for minikube, would make it much easier to use!  As far as using the addon standalone, I think that adding a config map w/ "mode" is the best solution for now.